### DISCUSSION TITLE
Kitchen Sink Example

### DISCUSSION THREAD

**Discussion by mfoitzik** [picocss/pico#550] [General]
I built a kitchen sink example for Pico CSS. I just wanted it for myself so I could see the style of all elements in one place. I am placing it here in case it is helpful for others. You can view it here: https://simplicitywebtools.com/pico-kitchensink/

**Comment by walterdavis** [comment]
That's really cool. I am noticing some elements spreading outside of the viewport left and right. Is there some outer-most container element that is missing? Or is this expected?



Safari version 17.5 (17618.2.12.111.5, 17618).

Walter

**Reply by mfoitzik** [reply]
I am guessing that it may be the button group (under Pico CSS Specific) causing this. I'll look into this a bit later. Thank you for the feedback.

**Comment by walterdavis** [comment]
Thanks. I did notice that at extremely wide window sizes, it does resolve, but at the moment (given the number of things stacked up horizontally in this part of the layout) there just isn't enough room and this area of the layout overflows beyond the right margin. I was able to scroll right and see it, but I didn't notice the overflow since my browser is very shy about scroll bars being visible.

Walter
